crates/sui-faucet/src/main.rs

```diff
@@ -92,6 +92,7 @@ async fn main() -> Result<(), anyhow::Error> {
         .route("/gas", post(request_gas))
         .route("/v1/gas", post(batch_request_gas))
         .route("/v1/status", post(request_status))
+        .route("/v1/gas_old", post(request_gas_v1_interface))
```

Comment:
The idea of the fallback is that it's controlled by a config flag on `faucet`, so we shouldn't need a new endpoint, instead, based on a config flag, `/v1/gas` would start serving batch requests without actually batching them.  This way, if there's an issue with the new implementation, we can fall back to the old one without having to tell any client to change which endpoitn they are calling.

Reply:
Changed this structure.